src/Tasks/LDAPGroupSyncTask.php

```diff
@@ -52,6 +57,8 @@ public function getTitle()
      */
     public function run($request)
     {
+        ini_set('max_execution_time', 900);
```

Comment:
`Environment::increaseTimeLimitTo(900);`

Reply:
Yeah so I tried that, but it doesn't work. `Environment::increaseTimeLimitTo()` and `Environment::increaseMemoryLimitTo()` increase only up to the maximum memory/time limit as defined (by default in php.ini).

So to do this 'properly' via Environment, I'd need to do:

```php
Environment::setMemoryLimitMax('1024M');
Environment::setTimeLimitMax(900);
Environment::increaseMemoryLimitTo();
Environment::increaseTimeLimitTo();
```

Which just seemed kinda overkill... but I can change that if you want?

Reply:
Yeah that does seem like overkill. They're tasks so its probably fine to leave it the way it is =)